src/rdkafka.h

```diff
@@ -5360,6 +5464,10 @@ typedef int rd_kafka_event_type_t;
 #define RD_KAFKA_EVENT_LISTCONSUMERGROUPOFFSETS_RESULT 0x8000
 /** AlterConsumerGroupOffsets_result_t */
 #define RD_KAFKA_EVENT_ALTERCONSUMERGROUPOFFSETS_RESULT 0x10000
+/* Describe User Scram Credentials*/
+#define RD_KAFKA_EVENT_DESCRIBEUSERSCRAMCREDENTIALS_RESULT 0x20000
+/* Alter User Scram Credentials*/
```

Comment:
Change comment to AlterUserScramCredentials_result_t and likewise for describe. See above.